src/app/api/services/dot-license/dot-license.service.ts

```diff
@@ -27,6 +76,33 @@ export class DotLicenseService {
         return this.getLicense().pipe(map((license) => license['level'] >= 200));
     }
 
+    /**
+     * Verifies if an url is an enterprise portlet and user has enterprise
+     *
+     * @param string url
+     * @returns Observable<boolean>
+     * @memberof DotLicenseService
+     */
+    canAccessEnterprisePortlet(url: string): Observable<boolean> {
+        return this.isEnterprise().pipe(
+            take(1),
+            map((isEnterpriseUser: boolean) => {
+                const urlMatch = this.checksIfEnterpriseUrl(url);
+                return urlMatch ? urlMatch && isEnterpriseUser : true;
+            })
+        );
+    }
+
+    private checksIfEnterpriseUrl(url: string): boolean {
+        const urlMatch = enterprisePorlets.filter((item) => {
+            return url.indexOf(item.url) >= 0;
```

Comment:
Can't you check this `===`? because this is matching partially so if we have two portlets with the same word in their url it will be a wrong match, right?